lumicks/pylake/tests/test_imaging_confocal/conftest.py

```diff
@@ -198,3 +200,31 @@ def kymo_h5_file(tmpdir_factory, test_kymo):
     )
 
     return mock_file.file
+
+
+@pytest.fixture(scope="module")
+def test_point_scan():
+    n_samples = 90
+    data = {c: np.random.poisson(15, n_samples) for c in ("red", "green", "blue")}
+
+    mock_file, metadata, stop = MockConfocalFile.from_streams(
+        start,
+        dt,
+        [],
+        [],
+        [],
+        infowave=np.zeros(data["red"].shape),
+        red_photon_counts=data["red"],
+        green_photon_counts=data["green"],
+        blue_photon_counts=data["blue"],
+    )
+    point_scan = PointScan("PointScan1", mock_file, start, stop, metadata)
+
+    reference = {
+        "data": data,
+        "timestamps": (np.arange(n_samples, dtype=np.int64) * dt) + start,
```

Comment:
Bracket is not needed.

```suggestion
        "timestamps": np.arange(n_samples, dtype=np.int64) * dt + start,
```